### PR TITLE
10/items open new link

### PR DESCRIPTION
**Pull request by hasanagh**
- wasn't sure about the "activity" demo mode so maybe we can fix in this MR 
- branched from 11

### CODE REVIEW

**Review by pyphilia — APPROVED** [review]
For me everything is okay, maybe I'll let @juancarlosfarah give more insights for this PR. I'll merge my own PR so that you can rebase, thank you @hasanagh !

**Comment by juancarlosfarah** [comment]
@hasanagh, I still see @pyphilia's commit mixed into this PR. Can you rebase?

**Comment by juancarlosfarah** [comment]
Any update on this @hasanagh ?

**Comment by hasanagh** [comment]
> Any update on this @hasanagh ?

Hey JC, sorry I just pushed now.

**Comment by pyphilia** [comment]
I'll rebase it on my changes, and tweak it a bit to use the api as well.

File: src/client/config/helpers.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const openInNewTab = (url) => {
+  console.log('url: ', url);
+  const newWindow = window.open(url, '_blank', 'noopener,noreferrer');
+  if (newWindow) newWindow.opener = null;
```
Comment (juancarlosfarah) [inline src/client/config/helpers.js:4]: Brackets around `if` statements, please. Can we force it with `eslint` / `prettier`?

File: src/client/config/helpers.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const openInNewTab = (url) => {
+  console.log('url: ', url);
```
Comment (juancarlosfarah) [inline src/client/config/helpers.js:2]: Remove console log here.

File: src/client/config/routes.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const COLLECTION_ROUTE = '/collections/*';
+export const buildSpaceRoute = (id = '*') => `/spaces/${id}`;
+export const HOME_ROUTE = '/*';
```
Comment (juancarlosfarah) [inline src/client/config/routes.js:3]: Does HOME need an `*`?

File: src/server.js [hunk 4]
```diff
@@ -58,6 +64,16 @@ const handleRender = (req, res, data) => {
   }
 };
 
+const handleSpaceRender = (req, res) => {
+  const id = req.path.split('/')[2];
```
Comment (juancarlosfarah) [inline src/server.js:68]: Comment or helper function to see what this is actually doing. Maybe `id` can be passed in a cleaner way.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks @hasanagh for cleaning this up. Just some quick comments and questions here.

**Pull request merged**